torch/testing/_internal/common_methods_invocations.py

```diff
@@ -960,6 +979,36 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
 
 # Operator database (sorted alphabetically)
 op_db: List[OpInfo] = [
+    UnaryUfuncInfo('abs',
```

Comment:
Since this adds an OpInfo for abs, this PR can remove other generated abs tests:

https://github.com/pytorch/pytorch/blob/869081961824dfb10ee944145c3a8d435571c53b/torch/testing/_internal/common_methods_invocations.py#L2104

https://github.com/pytorch/pytorch/blob/869081961824dfb10ee944145c3a8d435571c53b/test/test_torch.py#L6846